draft/customization/module-8/index.mdx

```diff
@@ -10,7 +10,7 @@ featuredImage: images/creatures/mantaray/mantaray-full@2x.png
 
 ## Outcome
 
-Understanding different components necessary for Ocean Protocol to operate including key concepts surrounding privacy preserving data sharing.
+From the very beginning, OcFrom the very beginning, Oc
```

Comment:
I guess this is not finished. Either way, don't bother working on anything in the draft folder